config/opensearch_dashboards.yml

```diff
@@ -152,4 +152,4 @@
 #vis_type_timeline.graphiteBlockedIPs: []
 
 # user input URL for customized logo 
-opensearchDashboards.branding.logoURL: "12324324"
```

Comment:
This is interesting, your dev_p0 branch is pretty identical to the upstream main. But the left side is saying you are changing something related to branding.logoURL.

Regardless, we probably want to keep branding_dev_p0 to be as close to prod ready as possible. So I think we should comment this out and have an empty string.

Reply:
Commented out and changed to empty string.